src/contracts/Chef.sol

```diff
@@ -54,6 +58,10 @@ abstract contract Chef is Ownable {
         return _rewardsActive;
     }
 
+    function getMetricToken() internal view virtual returns (MetricToken) {
```

Comment:
not sure if making this internal is necessary but thought only the contracts derived from this should be calling it